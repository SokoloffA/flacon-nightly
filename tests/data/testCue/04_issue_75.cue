PERFORMER "Michael Bond"
TITLE "Paddington's Finest Hour"
FILE "Paddingtons Finest Hour d1.flac" WAVE
REM DISCID 605aePKvRiNqUAth4gpwWnTIkfI-
REM DATE "2018"
REM GENRE "Audiobook"
REM NARRATOR "Stephen Fry"
  TRACK 01 AUDIO
    TITLE "01"
    INDEX 01 00:00:00
