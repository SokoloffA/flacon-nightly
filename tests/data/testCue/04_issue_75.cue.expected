[ DISC 01 / TRACK 01 ]
  FILE        = Paddingtons Finest Hour d1.flac
  ALBUM       = "Paddington's Finest Hour"
  TITLE       = 01
  DISCID      = 605aePKvRiNqUAth4gpwWnTIkfI-
  DATE        = 2018
  GENRE	      = Audiobook
  INDEX CD 00 = 00:00:00
  INDEX CD 01 = 00:00:00
  TRACKNUM    = 1
  TRACKCOUNT  = 1
  DISCNUM     = 1
  DISCCOUNT   = 1
